libevm/VMCalls.cpp

```diff
@@ -155,9 +155,8 @@ bool VM::caseCallSetup(CallParameters *callParams)
 	// "Static" costs already applied. Calculate call gas.
 	u256 requestedCallGas = *m_sp;
 	u256 maxAllowedCallGas = *m_io_gas - *m_io_gas / 64;
```

Comment:
As commented on an earlier version: I think it confusing to call this `maxAllowedCallGas` even if it is not an upper bound in all code paths.